tests/e2e/metrics/test_metrics.py

```diff
@@ -43,4 +43,5 @@ def test_metrics_on_dynatrace():
     response_json = response.json()
     assert 'totalCount' in response_json
     assert response_json['totalCount'] == 1
+    print(response_json)
```

Comment:
Should it be in this PR? 😄

Reply:
yes

Reply:
this will show full response when test will fail